### PR TITLE
Import from CoreBundle

### PR DESCRIPTION
**Pull request by core23**
TODO
- [ ] Wait for new CoreBundle major release
- [ ] Freeze CoreBundle
- [ ] Sync with CoreBundle

### CODE REVIEW

File: src/Bridge/Symfony/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        $rootNode = $treeBuilder->root('sonata_form');
+
+        $this->addFlashMessageSection($rootNode);
+
+        $rootNode
+            ->children()
+                ->arrayNode('form')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('mapping')
```
Comment (covex-nn) [inline src/Bridge/Symfony/DependencyInjection/Configuration.php]: Form mapping was deprecated in `3.x`. May be this block should be removed from here?

Comment (OskarStark) [reply src/Bridge/Symfony/DependencyInjection/Configuration.php]: But this is still used in `3.x` isn't it?

Comment (core23) [reply src/Bridge/Symfony/DependencyInjection/Configuration.php]: Removed the mapping, is the node isn't used anymore.

File: composer.json [hunk 2]
```diff
@@ -21,10 +21,20 @@
     ],
     "require": {
         "php": "^7.1",
```
Comment (core23) [inline composer.json]: Can we use PHP 7.1 here? This is a version break, if we require this in the CoreBundle stable version

File: src/Bridge/Symfony/Bundle/SonataFormBundle.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ */
+
+namespace Sonata\Form\Bridge\Symfony\Bundle;
+
+use Sonata\Form\Bridge\Symfony\DependencyInjection\SonataFormExtension;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+final class SonataFormBundle extends Bundle
+{
+    public function getPath()
```
Comment (greg0ire) [inline src/Bridge/Symfony/Bundle/SonataFormBundle.php]: Please add type hints/final everywhere you can

File: composer.json (hunk 2 above)
Comment (greg0ire) [reply composer.json]: I think it should be ok now that Ubuntu supports 7.1 . We've been bashed in the past for doing this, but it was already for 7.1, and we are not requiring 7.2.

Comment (kunicmarko20) [reply composer.json]: On master, we can, but on 1.x it should be same as core 3.x

**Comment by core23** [comment]
Duplicate of https://github.com/sonata-project/form-extensions/pull/7

**Pull request closed without merging**